src/containers/Footer/index.tsx

```diff
@@ -1,40 +1,9 @@
-import styled from 'styled-components';
-
 export const Footer = () => {
   return (
-    <FooterContainer>
+    <>
```

Comment:
use a `footer` tag instead of a react fragment

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/commit/9b5c51f0ad472ae9a2584c0f0d364b6e6f83e685